### PR TITLE
In 139870345 verify session tokens

### PR DESCRIPTION
**Pull request by saraOSI**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -68,4 +68,10 @@
     example.run
     WebMock.allow_net_connect!
   end
+
+  config.around(:example, accessibility: false) do |example|
+    Capybara.default_driver = :selenium
```
Comment (NateOSI) [inline spec/spec_helper.rb]: This should technically set the default driver back to what it was
```
tmp_driver = Capybara.default_driver
Capybara.default_driver = :selenium
example.run
Capybara.default_driver = tmp_driver
```

File: spec/features/api_response_spec.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  scenario 'API returns a 403', accessibility: false do
+    stub_request(:get, api_screenings_path).and_return(
+      body: { screenings: [] }.to_json,
+      status: 200,
+      headers: { 'Content-Type' => 'application/json' }
+    )
+    visit root_path
+    base_url = ENV.fetch('AUTHENTICATION_URL').chomp('/')
+    redirect_url = CGI.escape("#{page.current_url.chomp('/')}#{screening_path(screening.id)}")
+    login_url = "#{base_url}/authn/login?callback=#{redirect_url}"
```
Comment (NateOSI) [inline spec/features/api_response_spec.rb]: is it worth putting login_url in a webmock helper? like the api_screenings_path and api_people_path etc?

File: spec/repositories/security_repository_spec.rb [hunk 3]
```diff
@@ -3,44 +3,32 @@
 
 describe SecurityRepository do
   describe '.login_url' do
-    before do
-      allow(ENV).to receive(:fetch)
-        .with('AUTHENTICATION_URL')
-        .and_return('http://authentication')
-    end
+    let(:authentication_login_url) { Rails.configuration.intake[:authentication_login_url] }
```
Comment (NateOSI) [inline spec/repositories/security_repository_spec.rb]: Rails.configuration.intake[:authentication_login_url] should be stubbed here

File: spec/spec_helper.rb (hunk 1 above)
Comment (saraOSI) [inline spec/spec_helper.rb]: Will fix!

**Pull request closed without merging**